src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+  width: 96px;
+
+  list-style: none;
+
+  &__star {
+    background: no-repeat center url(../images/star.svg);
+    height: $star-size;
+    width: $star-size;
+  }
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background: no-repeat center url(../images/star-active.svg);
+}
```

Comment:
- Use nesting
- It should be the same functionality as in the stars task, if the modifier is `stars--3` you should show 3 active stars and so on